torch/csrc/jit/passes/python_print.cpp

```diff
@@ -489,12 +490,34 @@ struct PythonPrintPass {
     stmt << end;
   }
 
+  template <typename T>
```

Comment:
What!? This will be really confusing when debugging if the order of loop carried dependencies now becomes a property of the debug names given to them. It is probably possible to eliminate the jitter that this is probably fixing in another way.

Reply:
It is also very possible this is a bug. `useOf` for single-use things is not an identifier and it is possible they have side effects.

Reply:
This is reordering the order in which useOf is printed. The text that useOf contains can have side effects in it.

Reply:
Using debug names (which can be anything) is different from using variable names (which are not modified by any passes and are exactly as the user wrote). Change debug names should never change the order of things in the graph.